javatests/arcs/core/data/AnnotationTest.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [Annotation]. */
+@RunWith(JUnit4::class)
+class AnnotationTest {
+    @Test
+    fun from_string_no_param() {
```

Comment:
This should be camelCase, with underscores separating different sections:

```
fromString_noParam
```

Reply:
done